ui/partials/perf/dashboard.html

```diff
@@ -1,37 +0,0 @@
-<div class="container-fluid">
-  <div class="alert alert-danger" ng-show="revisionNotFound" role="alert">
-    Couldn't find revision {{revision}} for {{selectedRepo.name}}
-  </div>
-  <div ng-if="!revision">
-    <h1>{{header}}</h1>
-    <p>{{descP1}}</p>
-    <p>{{descP2}} <a href="{{linkUrl}}">{{linkDesc}}</a></p>
-    <div class="form-group">
-      Branch to compare:
-      <select ng-change="globalOptionsChanged(selectedRepo, selectedTimeRange)"
-              ng-model="selectedRepo"
-              ng-options="repo.name for repo in projects">
-      </select>
-      Time range to sample:
-      <select ng-model="selectedTimeRange"
-              ng-options="timeRange.text for timeRange in timeRanges track by timeRange.value"
-              ng-change="globalOptionsChanged(selectedRepo, selectedTimeRange)">
-      </select>
-    </div>
-  </div>
-  <div ng-if="dataLoading">
-    Loading all results, please wait a minute...
-    <img src="../../img/dancing_cat.gif" />
-  </div>
-  <div ng-if="!dataLoading && !revisionNotFound">
-    <ph-compare-table
-       base-title={{baseTitle}}
-       new-title={{variantTitle}}
```

Comment:
Similarly, I believe all `newTitle` and `baseTitle` references can now also be removed, since the standard compare views don't use them.

Reply:
It seems they're used in the `trendtable.html` and `comparetable.html` partials - either of which the phCompareTable component uses so leaving in for now.

Reply:
The `baseTitle` and `newTitle` variables allowed for overriding of the headings on those pages, however nothing overrides them any more, so they are always at default values. As such the default values can be moved from here:
https://github.com/mozilla/treeherder/blob/debac0c5c2e2c4623a1582b57a2afdbf9340d242/ui/js/components/perf/compare.js#L30-L35

...to the templates, and then `baseTitle` and `newTitle` removed.

Reply:
I see - thanks!